src/components/accordion/accordion.spec.tsx

```diff
@@ -615,7 +628,7 @@ describe("AccordionGroup", () => {
     container = null;
   });
 
-  testStyledSystemMargin((props) => (
+  testStyledSystemMargin((props: AccordionProps) => (
```

Comment:
```suggestion
  testStyledSystemMargin((props) => (
```

This change will lead to a type error after my `test-utils` refactor gets merged

Reply:
don't worry, I'm putting this task as blocked and I will sort this out after your PR is merged